Ports/SDL2_sound/package.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=SDL_sound
-version=git
-_commit=df3fc779774c2c5dc1147239da1af858c88f1a74
-workdir=SDL_sound-${_commit}
-useconfigure=true
-depends=("SDL2")
+port='SDL2_sound'
```

Comment:
Surely, there are ports that rely on `SDL_sound`, right? Those ports dependencies should be adapted as well.

Reply:
```
[lightspot21@Workbox-MK2 Ports]$ rg SDL_sound
SDL_sound/package.sh
2:port='SDL_sound'
7:files="https://www.icculus.org/SDL_sound/downloads/${port}-${version}.tar.gz ${port}-${version}.tar.gz 3999fd0bbb485289a52be14b2f68b571cb84e380cc43387eadf778f64c79e6df"

gltron/package.sh
8:depends=("libpng" "glu" "SDL_sound" "sdl12-compat" "zlib")
13:    "LIBS=-lm -lSDL_sound"

SDL2_sound/package.sh
5:workdir="SDL_sound-${_commit}"
8:files="https://github.com/icculus/SDL_sound/archive/${_commit}.zip ${_commit}.zip bf655a03ab96a49c4140e19135433d62893c124330955e85e3dfddbe9963bac2"

SDL2_sound/patches/ReadMe.md
1:# Patches for SDL_sound on SerenityOS

AvailablePorts.md
199:| [`SDL2_sound`](SDL2_sound/)                         | SDL2\_sound (Abstract soundfile decoder add-on for SDL2)        |                          | https://github.com/icculus/SDL_sound                                           |
202:| [`SDL_sound`](SDL_sound/)                           | SDL_sound (Abstract soundfile decoder add-on for SDL 1.2)       | 1.0.3                    | https://www.icculus.org/SDL_sound/                                             |
```

Seems like nothing else apart from gltron itself needs SDL_sound, actually

Reply:
Then that dependency should be renamed in this commit and only be replaced later on.

Reply:
Done